calculator/src/main/java/com/market/banica/calculator/model/Pair.java

```diff
@@ -0,0 +1,11 @@
+package com.market.banica.calculator.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class Pair  <T,K>{
+    private T fst;
+    private K snd;
```

Comment:
Names could be more descriptive.

Reply:
As it is a generic class with several use cases in the code, field names have to be generic too. I applied Sun's Pair class convention fst for the first and snd for the second. Do you mean to replace it with "first", respectively "second"?

Reply:
If that's the reason , then it is ok to leave it as it is.

Reply:
Why not use the already available Pair class located in javafx.util ?

Reply:
the name of the variables should be descriptive, at least a word, first and second sound better
also, I can see that the class is not formatted

Reply:
@gaantonova It is no longer part of the Java JDK,  you need to download the JavaFX SDK to your machine 
@zaraandreevaa fixed